shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -46,6 +47,12 @@
 @NotThreadSafe
 public class TfsShell implements Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
+  private static final HashMap<String, String[]> CMD_ALIAS = new HashMap<String, String[]>() { {
+      put("chgrpr", new String[] {"chgrp", "-R"});
```

Comment:
I think it's no longer necessary to keep *RecursiveCommand classes since they are deprecated. Using a "aliases" concept within TfsShell seems clearer to me: the benefits are 1) some unused classes get removed; 2) each command logic is now in a single class.